File: problems/baekjoon/09_2차원 배열/2566_최댓값/jaeseok.py
board = []

for _ in range(9):
  board.append(list(map(int, input().split())))

row = 0
col = 0
MAX = -99999;

for r in range(9):
  for c in range(9):
    if board[r][c] > MAX:
      MAX = board[r][c]
      row = r + 1
      col = c + 1
print(MAX)
print(row, col)

File: problems/baekjoon/09_2차원 배열/2566_최댓값/jaeseok.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : 'example.txt';
let input = fs.readFileSync(filePath).toString().split('\n');

let board = []
for (let i = 0; i < 9; i++) {
  board.push(input[i].split(' ').map(Number));
}

// let flatBoard = board.flat();
let flatBoard = [].concat(...board);

let maxNum = Math.max(...flatBoard);
console.log(maxNum);

let maxNumIdx = flatBoard.indexOf(maxNum);
let row = Math.floor(maxNumIdx/9) + 1;
let col = (maxNumIdx%9) + 1;
console.log(row, col);